sky/shell/platform/ios/sky_surface.mm

```diff
@@ -112,6 +115,14 @@ - (instancetype)initWithShellView:(sky::shell::ShellView*)shellView {
 
     _shell_view.reset(shellView);
     self.multipleTouchEnabled = YES;
+
+    [[NSNotificationCenter defaultCenter] addObserver:self
```

Comment:
Generally a good practice to get rid of these registrations (and collect the viewport metrics instance above) in the dealloc method. I realize the surface is around for a long time though.